### DISCUSSION TITLE
MS Access Versions

### DISCUSSION THREAD

**Discussion by space-man-mat** [joyfullservice/msaccess-vcs-addin#362] [General]
I cannot see a list of MS Access versions that are supported by this plugin listed anywhere.

Can you please advise if MS Access 2013 is supported by the current plugin version?

Also we will probably be upgrading at some stage as support will end next year, so what is the outlook going forward?

**Comment by joyfullservice** [comment]
This add-in runs on Access 2010 and newer, so yes, 2013 is supported, all the way up to the latest versions of Microsoft Access. Thanks for pointing out the fact that this is missing on the home page readme file. I have updated the page to list the supported versions.
